README.md

```diff
@@ -28,11 +27,13 @@ You have to create a token with the `read:org` access.
 The following command allows you to get information on your organization
 
 ```
-yarn start
+yarn start <organization_name>
 ```
 
 The following command allows you to get insights from the information
 
 ```
-yarn stats
+yarn stats <organization_name>
 ```
+
+You can add the organisazation name in the .env file. In this case use the key `GITHUB_ORGA`. If the two chose are set the env variable will be take.
```

Comment:
Two small changes and we are done 👍 
`If the two chose` => `If both choices` 
`take` => `taken`